.github/workflows/ci-cd-pipeline.yml

```
@@ -2,33 +2,33 @@ name: Template App CI/CD Pipeline
+  #   name: Setup Mint
+  #   uses: ./.github/workflows/setup-mint.yml
+  #   permissions:
+  #     contents: read
 
   # コード品質チェック
   code-quality:
     name: Code Quality Check
-    needs: setup-mint
+    # needs: setup-mint
     uses: ./.github/workflows/code-quality.yml
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)